packages/components/src/date-time/time.js

```diff
@@ -62,6 +63,12 @@ class TimePicker extends Component {
 		}
 	}
 
+	changeDate( newDate ) {
```

Comment:
Might be nice for some docs, even if there's plenty of prior art of no documentation in this component.